src/caches/ssprk_caches.jl

```diff
@@ -169,6 +169,97 @@ function alg_cache(alg::SHLDDRK52,u,rate_prototype,uEltypeNoUnits,uBottomEltypeN
   SHLDDRK52Cache(u,uprev,k,tmp,fsalfirst,tab)
 end
 
+@cache mutable struct SHLDDRK_2NCache{uType,rateType,TabType} <: OrdinaryDiffEqMutableCache
+  u::uType
+  uprev::uType
+  k::rateType
+  tmp::uType
+  fsalfirst::rateType
+  tab::TabType
+  step::Int
+end
+
+mutable struct SHLDDRK_2NConstantCache{T1,T2} <: OrdinaryDiffEqConstantCache
+  α21::T1
+  α31::T1
+  α41::T1
+  α51::T1
+  β11::T1
+  β21::T1
+  β31::T1
+  β41::T1
+  β51::T1
+  c21::T2
+  c31::T2
+  c41::T2
+  c51::T2
+
+  α22::T1
+  α32::T1
+  α42::T1
+  α52::T1
+  α62::T1
+  β12::T1
+  β22::T1
+  β32::T1
+  β42::T1
+  β52::T1
+  β62::T1
+  c22::T2
+  c32::T2
+  c42::T2
+  c52::T2
+  c62::T2
+
+  step::Int
+end
+
+function SHLDDRK_2NConstantCache(T1,T2)
+  α21 = T1(-0.6051226)
```

Comment:
these coefficients are not high accuracy. Do they not provide more digits?

Reply:
There are only these many digits provided.